parlai/core/utils.py

```diff
@@ -424,7 +424,7 @@ def map_predictions(cls, predictions, valid_inds, batch_reply, observations, dic
            using valid_inds.
            report_freq -- how often we report predictions
         """
-        predictions = predictions.cpu()
+        predictions = predictions.cpu().data
```

Comment:
does there need to be some check whether you are cpu or gpu or this doesnt affect performance for cpu?

Reply:
I'm pretty sure this is a no-op if the variable is already on CPU; torch docs say that the method `Returns a CPU copy of this tensor if it's not already on the CPU`